### PR TITLE
Basic portfolio

### PR DESCRIPTION
**Pull request by deeptivaidy**
I included a basic personal portfolio with a short description of who I am and a few of my favorite things.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -3,16 +3,62 @@
...
+                <button onclick="addRandomGreeting()">Click me for a Quote!</button>
+                <div id="greeting-container"><i></i></div>
+            </div>
+            
+        </div>
+        <div id="content">
+            <h1><b>Deepti Vaidyanathan</b></h1>
+            <p>I am a 3rd year Computer Science major at the Georgia Institute of Technology 
+                with threads in Intelligence and Information/Internetworks. I am interested 
+                in data science, information security, and artificial intelligence espeicially
```
Comment (jayceb04) [inline portfolio/src/main/webapp/index.html:26]: especially

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,42 @@
+body {
+    background-image: url("/images/background.jpg");
+    background-repeat: repeat-x;
+}
+#profile {
+    /* padding:0px; */
```
Comment (travis-miller) [inline portfolio/src/main/webapp/style.css:6]: It is usually better to delete commented code when committing it.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,42 @@
...
+}
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    background-color: rgba(253, 253, 243, 0.5);
+    padding: 20px;
+    width: 650px;
+    border: 5px solid darkolivegreen;
+    /* float: left; */
```
Comment (travis-miller) [inline portfolio/src/main/webapp/style.css:31]: Same here...

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,42 @@
...
+    margin-left: 400px;
+    /* margin-right: 200; */
```
Comment (travis-miller) [inline portfolio/src/main/webapp/style.css:33]: and here.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (deeptivaidy) [reply portfolio/src/main/webapp/index.html]: Thanks for catching that! I fixed it in the most recent commit

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (deeptivaidy) [reply portfolio/src/main/webapp/style.css]: Thanks for letting me know! I removed those comments in the most recent commit, I'll keep this in mind in the future

**Pull request merged**